app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -33,10 +33,12 @@
   }
 }
 
-@include govuk-grid-row($class: "gem-c-feedback__grid-row");
+.gem-c-feedback__grid-row {
+  @extend %govuk-grid-row;
```

Comment:
V3 changelog says `You can no longer use the govuk-grid-row mixin` - that's not this, is it? I haven't checked the review app (it's not working atm).

Reply:
Yes, it's that one. This was removed in a following commit.